src/WebJobs.Script/Workers/Rpc/InvocationException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
+{
+    public class InvocationException : Exception
+    {
+        public InvocationException(string result, string message, string stack)
```

Comment:
Is there a way we can make sure that the namespace isn't shown in these errors i.e. `InvocationException` vs. `Microsoft.Azure.WebJobs.Script.Workers.Rpc.InvocationException`

Honestly I don't think we can remove the  `Microsoft.Azure.WebJobs.Script.Workers.Rpc` part unless we actually change the namespace, but just curious if it is possible. Just want to ensure to users it's clear that it's their issue